ext/curl/tests/curl_version_error_001.phpt

```diff
@@ -1,15 +1,18 @@
 --TEST--
-curl_version(): error conditions
+curl_version(): error condition
 --SKIPIF--
 <?php
 if (!extension_loaded('curl')) die('skip curl extension not available');
 ?>
 --FILE--
 <?php
-curl_version(CURLVERSION_NOW);
-curl_version(0);
-?>
---EXPECTF--
-Deprecated: curl_version(): The $version parameter is deprecated in %s on line %d
 
-Warning: curl_version(): $version argument ignored in %s on line %d
+try {
+    curl_version(1);
+} catch (ArgumentCountError $exception) {
+    echo $exception->getMessage() . "\n";
+}
+
+?>
+--EXPECT--
+curl_version() expects exactly 0 parameters, 1 given
```

Comment:
Should we drop this one? Looks like a pure ZPP test for me.

Reply:
Yeah, that's a good idea :)